action-constraints.txt

```diff
@@ -55,7 +55,7 @@ google-auth==2.38.0
     #   google-cloud-kms
 google-cloud-kms==3.2.2
     # via securesystemslib
-googleapis-common-protos==1.66.0
+googleapis-common-protos==1.67.0rc1
```

Comment:
always a good sign to get rc dependencies...

Reply:
weirdly I don't get this same result locally

Reply:
Interesting. How do you want to proceed here?

Reply:
Not sure:
* I can see this happens only when the dependency (in two of our dependencies) is `googleapis-common-protos<2.0.0dev` -- this seems weird but I'm not sure why this is related
* I still have no idea why this only happens in GitHub Actions: locally I have the same Python version, same pip-tools version... I still can't reproduce

Reply:
Oh I got it now: 
* pip 23.3 does not reproduce
* pip 25.0.1 does reproduce

Reply:
https://github.com/pypa/pip/issues/13217

Reply:
as pip devs pointed out, pip was just fixing the previous incorrect resolution.

Real issue is google libraries using "googleapis-common-protos<2.0.0dev" style requirements: I've filed this internally. For now we might have to do some manual fixes -- but we can let these collect for a week or two

Reply:
Great find.

Reply:
googleapis-common-protos made an actual release so this specific issue has gone away: it will appear again though